charts/dex/values.yaml

```diff
@@ -328,3 +328,21 @@ networkPolicy:
   #    ports:
   #      - port: 636
   #        protocol: TCP
+
+# -- List of extra manifests to deploy. Will be passed through `tpl` to support templating
+extraObjects: []
```

Comment:
I noticed this is called `extraManifests` in some cases. Is there some sort of consensus how this should be called? (TBH `extraManifests` sounds somewhat better to me, but I have no objective reason for that preference, although you called it "manifests" as well in the changelog and the documentation)

Reply:
🤷‍♂️ 😄
I took `extraObjects` since I am a maintainer of [argo-helm](https://github.com/argoproj/argo-helm) and we use this.

Reply:
Fair enough. 😄